AxonDeepSeg/segment.py

```diff
@@ -154,20 +142,13 @@ def segment_folders(path_testing_images_folder, path_model,
         # Read image for conversion
         img = ads.imread(str(path_testing_images_folder / file_))
 
-        # Generate tmpfile for segmentation pipeline
-        fp = open(path_testing_images_folder / '__tmp_segment__.png', 'wb+')
-
         img_name_original = file_.stem
 
-        ads.imwrite(fp, img, format='png')
```

Comment:
Where'd all this deleted code go?

Reply:
To answer your question here is the workflow of what old code did
1. Create a `__tmp_segment__.png` file for each of the image the user wants to segments. The `__tmp_segment__.png` file is nothing but the image.
2. Write this `__tmp_segment__.png` file in the working directory.
3. Call the function axon_segmentation to segment the image
4. Finally, after the image has been segmented, remove the `__tmp_segment__.png` from the working directory.

Now, here is why I deleted this code:
1. First of all, I don't understand why are we creating a temp file `__tmp_segment__.png` file when we can directly work on the image we wish to segment.
2. Second, when we segment each image by creating a temp file, the acquisition_name variable's naming is not readable and intuitive. For eg. let's say the image we wish to segment is **test.png**, so the acquisition name would be  `__tmp_segment__.png` according to the deleted code. But shouldn't it be **test.png**?

Hence, I deleted this code.